src/modules/inventory/tests/inventoryImp/CMakeLists.txt

```diff
@@ -8,4 +8,6 @@ target_link_libraries(inventoryimp_unit_test PRIVATE
     GTest::gtest_main
     GTest::gmock
     GTest::gmock_main)
+if(NOT WIN32)
 add_test(NAME InventoryImpTest COMMAND inventoryimp_unit_test)
+endif()
```

Comment:
this means that this test is not available under win32? if it's so, shouldn't we directly avoid to add this directory on a directory above?

Reply:
This is temporary. These tests **must** be present and passed. But since this is still with several functions mocked with empty functions, they are not able to run.

Reply:
We will add a ToDo commentary to remind to restore these tests once mocked functions are implemented